sdk/keyvault/keyvault-keys/src/transformations.ts

```diff
@@ -53,3 +60,69 @@ export function getKeyFromKeyBundle(
 
   return resultObject;
 }
+
+/**
+ * @internal
+ * Shapes the exposed {@link DeletedKey} based on a received KeyItem.
+ */
+export function getDeletedKeyFromDeletedKeyItem(keyItem: DeletedKeyItem): DeletedKey {
+  const parsedId = parseKeyVaultKeyId(keyItem.kid!);
+  const attributes = keyItem.attributes || {};
+
+  const abstractProperties: KeyProperties = {
```

Comment:
nit: this looks really similar to what gets returned from `getKeyPropertiesFromKeyItem` - is it possible to call that function instead?

`const abstractProperties = getKeyPropertiesFromKeyItem(keyItem)`

Reply:
You're correct! I've pushed an update following your suggestions 👍